src/Library/Logger/DistLogSink.cpp

```diff
@@ -9,6 +9,15 @@ void DistLogSink::write(const LogCategory &category, LogLevel level, std::string
     }
 }
 
-void DistLogSink::addLogSink(std::unique_ptr<LogSink> logSink) {
+std::function<void()> DistLogSink::addLogSink(std::unique_ptr<LogSink> logSink) {
```

Comment:
Just add a `removeLogSink(LogSink *)` method

Reply:
I removed entirely the returned function. I'm dealing with the need of unregistration differently so I don't need any `removeLogSink` for now.